client/src/components/Views/TripSidePanelSingle.js

```diff
@@ -37,13 +37,13 @@ export function TripSidePanelSingle() {
   const renderStages = useMemo(() => {
     if (!dayDetails) return <></>
 
-    return dayDetails['stages'].map((stage, index) => {
-      const toColor = `to-${getColorName(dayDetails['color'])}-400/${
-        index * 10 + 10
-      }`
+    return dayDetails.map((stage, index) => {
+      const toColor = `to-${getTailwindName(
+        dayDetails[0]['colorNumber'],
```

Comment:
same as above

Reply:
extracted a variable for clarity, as we discussed in discord that the `[0]` is necessary